Makefile

```
@@ -132,14 +132,35 @@ preflight:
-.PHONY: data feature research resolve-comments unit-tests integration-tests analyze-architecture
+.PHONY: data seed-local seed-remote feature research resolve-comments unit-tests integration-tests analyze-architecture
 
-data: start-firebase-emulators
-	@echo "Seeding emulator data using standalone seeder..."
+data: seed-local
+
+seed-local: start-firebase-emulators
+	@echo "Seeding local emulator data using standalone seeder..."
 	@dart run tool/seed_database.dart
 
+seed-remote:
```

checkmake (0.3.2)

[warning] Target body for "seed-remote" exceeds allowed length of 5 lines (6).

(maxbodylength)